packages/reactive_ble_platform_interface/lib/src/model/characteristic_instance.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:meta/meta.dart';
+
+import 'uuid.dart';
+
+@immutable
+class CharacteristicInstance {
+  /// Unique uuid of the specific characteristic
+  final Uuid characteristicId;
+
+  /// When empty, the first characteristic with [characteristicId] is used
```

Comment:
When you say empty you are talking about empty string? Sorry, got a little bit confused with it.

Reply:
And if so, why decided to go with empty string instead of nullability?

Reply:
That comment isn't relevant/correct anymore. I'll replace it with an explanation of what instance id is.